Cargo.toml

```diff
@@ -69,6 +69,7 @@ toml_edit = { version = "0.4.0", features = ["easy"] }
 url = "2.1.1"
 ureq = { version = "1.5.1", default-features = false, features = ["tls", "json", "socks"] }
 pathdiff = "0.2"
+clap-cargo = "0.6.1"
```

Comment:
I just released 0.7 to add the `-p` flag which was missing.  We'll want this to not have a regression in people's workflows

Reply:
@epage we probably should also add `structopt...conflicts_with` to `clap-cargo`, as currently nothing prevents you from passing `--workspace`, `--all` & `-p` simultaneously.

Reply:
These are not conflicting in [cargo](https://github.com/rust-lang/cargo/blob/master/src/cargo/util/command_prelude.rs#L31) so by using `clap-cargo`, we'll be better aligning in our behavior in this way as well.